EpikCord/client/client_application.py

```diff
@@ -106,53 +104,54 @@ async def edit_global_application_command(
         name: Optional[str] = None,
         description: Optional[str] = None,
         options: Optional[List[AnyOption]] = None,
-        default_permissions: Optional[bool] = None,
+        default_member_permissions: Optional[Permissions] = None,
     ):
-        payload = {}
-        if name:
-            payload["name"] = name
-        if description:
-            payload["description"] = description
+        payload: Dict[str, Optional[Union[List[discord_typings.ApplicationCommandOptionData], str, int]]] = {
+            "name": name,
+            "description": description,
+        }
         if options:
             payload["options"] = [option.to_dict() for option in options]
-        if default_permissions:
-            payload["default_permissions"] = default_permissions
+        if default_member_permissions:
+            payload["default_member_permissions"] = default_member_permissions.value if isinstance(default_member_permissions, Permissions) else default_member_permissions
```

Comment:
The length of the line... **171**
![image](https://user-images.githubusercontent.com/53050011/194423331-d745efd5-d6b8-4d40-92ee-a61657270dd8.png)

Reply:
Surely Black would fix that?

Reply:
ye but man, dont rely solely on the formatter

Reply:
Tbh I didn't even see the line length.